src/test/java/com/rtb/projectmanagementtool/project/servlets/DeleteProjectServletTest.java

```diff
@@ -0,0 +1,58 @@
+package com.rtb.projectmanagementtool.project;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.dev.LocalDatastoreService;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import java.io.*;
+import java.util.Collections;
+import javax.servlet.http.*;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.*;
+
+public class DeleteProjectServletTest extends Mockito {
+  private HttpServletRequest request;
+  private HttpServletResponse response;
+  private DatastoreService datastore;
+  private ProjectController projectController;
+  private DeleteProjectServlet servlet;
+
+  private final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(
+          new LocalDatastoreServiceTestConfig()
+              .setAutoIdAllocationPolicy(LocalDatastoreService.AutoIdAllocationPolicy.SEQUENTIAL));
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    request = mock(HttpServletRequest.class);
+    response = mock(HttpServletResponse.class);
+    datastore = DatastoreServiceFactory.getDatastoreService();
+    projectController = new ProjectController(datastore);
+    servlet = new DeleteProjectServlet();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  @Test
+  public void deleteProject() throws Exception {
+    projectController.addProject(new ProjectData("Project", "Desc", 0l));
```

Comment:
Watch out with these since this will mean if your controller addProject breaks (or if you update the logic), your servlet tests could also break. One alternative you could look into is if we can use mockito to mock the controller, so we don't actually invoke the controller when testing the servlet.